tests/Integration/Controller/ShowProductActionTest.php

```diff
@@ -27,9 +27,10 @@ public function itDisplaysAProduct(): void
         $client = $this->initializeClientWithSession([
             'pim_url' => 'https://example.com',
             'akeneo_pim_access_token' => 'random_access_token',
+            'akeneo_pim_catalog_id' => 'catalog_store_us_id',
```

Comment:
Real catalog ids are uuids. I'm wondering what's the best between readability vs real life usage here (because we are in a demo app for external developers). We are in the tests so maybe my comment is not relevant. What do you think?

Reply:
And also the catalog id used here is not the same as in the JSON response links (next, self, ...).

Reply:
There were already catalog_store_us_id mentions and files so i followed it, i think it's ok and makes the reading and navigation between tests & fixtures easier.

I will change the bad catalogId on the fixtures.

Reply:
Are the changes ok for you @masacc ?

Reply:
:+1:

Reply:
I always advocate using test data that is closest to the real data. Tests have 2 main purposes: to automatically verify that the code does what it should do, and to document the code. If you put test data that doesn't match the real data both are in danger. You can have tests that are valid only because the data you use is not the right one and you wrote the test after writing the code. And you make the documentation of the code less readable because it can be thought that the real data is supposed to be like your test data.

Reply:
I replaced every catalog_store_us_id and catalog_store_fr_id with real uuids